### PR TITLE
Add docker_compose_path option

### PR DESCRIPTION
**Pull request by Mossie93**
## Description

Let user specify path to the compose path.

### CODE REVIEW

File: lib/capistrano/tasks/docker/compose.rake [hunk 1]
```diff
@@ -34,7 +34,7 @@ namespace :docker do
       task :build do
         on roles(fetch(:docker_role)) do
           within release_path do
-            execute :"docker-compose", compose_build_command
+            execute :"docker-compose -f #{fetch(docker_compose_path)}", compose_build_command
```
Comment (rindek) [inline lib/capistrano/tasks/docker/compose.rake]: @Mossie93 if you can please move the -f option into the `compose_build_command` method, and similiarly in other places as well

File: lib/capistrano/tasks/docker/compose.rake [hunk 2]
```diff
@@ -52,7 +52,6 @@ namespace :docker do
       task :stop do
         on roles(fetch(:docker_role)) do
           within release_path do
-            execute :"docker-compose", compose_stop_command
```
Comment (rindek) [inline lib/capistrano/tasks/docker/compose.rake:55]: I think you accidentaly removed something :)

File: lib/capistrano/tasks/docker/compose.rake [hunk 3]
```diff
@@ -81,6 +81,7 @@ namespace :docker do
   def compose_build_command
     cmd = ["build"]
     cmd << fetch(:docker_compose_build_services) unless fetch(:docker_compose_build_services).nil?
+    cmd.unshift("-f #{fetch(:docker_compose_path)}")
```
Comment (rindek) [inline lib/capistrano/tasks/docker/compose.rake]: The -f option, according to docker-compose, must be added just after docker-compose command,

For instance: `docker-compose -f {docker_compose_path} -p {docker_compose_project_name} build {docker_compose_build_services}`

So this unshift should be added at the very last lines

File: lib/capistrano/tasks/docker/compose.rake (hunk 2 above)
Comment (Mossie93) [inline lib/capistrano/tasks/docker/compose.rake:55]: Uh, sorry :(

**Comment by rindek** [comment]
@Mossie93 looks great, thanks for that :-)

**Pull request merged**